backend/src/layer/data/swka_parser/html_parser.rs

```diff
@@ -151,149 +140,146 @@ impl HTMLParser {
     ///
     /// ```
     /// use crate::mensa_app_backend::layer::data::swka_parser::html_parser::HTMLParser;
-    /// let canteen_data = HTMLParser::transform(include_str!("./tests/test_normal.html"));
+    /// let canteen_data = HTMLParser::transform(include_str!("./test_data/test_normal.html"));
```

Comment:
maybe you want to unwrap the result returned here so the doctest fails accordingly